cleanlab/datalab/internal/issue_manager/data_valuation.py

```diff
@@ -37,7 +37,22 @@
 
 
 class DataValuationIssueManager(IssueManager):
-    """Manages data sample with low valuation."""
+    """
+    Manages data sample with low valuation.
+
```

Comment:
Can we change in this file:

```
DEFAULT_THRESHOLDS = 0
```

I think that makes more sense than 1e-6. I.e. datapoints that contribute negative value are considered issues in the default setting.

Reply:
As discussed before, the data valuation score will be transformed to [0,1] to be the same as other issue scores. So I think 0 doesn't make sense. Maybe I can do more experiments to figure out a better threshold that can work for most datasets.

Reply:
Ah yes, sorry I forgot about the transformation. Let's keep threshold as is. 

Can you add in the issue type description 1 sentence about how we mathematically transform the raw Data Shapely score to the interval 0-1?

Any additional experiments to help us find a good default threshold would be much appreciated!  Ideally our default threshold should capture most negatively-contributing and useless datapoints

Reply:
OK, I will do some additional experiments to find a better default threshold later.